### DISCUSSION TITLE
Rozwój programu.

### DISCUSSION THREAD

**Discussion by RybarskiDominik** [RybarskiDominik/GML-2021#7] [General]
Plany:
- moduł zawiadomienia
- moduł protokoły ustalenia
Gotowe prototypy lecz potrzebne dalsze prace nad implementacją.

- poprawa ikony programu
- poprawa wyglądu aplikacji

Sugestie mile widziane:

**Comment by Kamila111111** [comment]
Witam, przetestowałam program i mam kilka sugestii, które myślę że w znacznym stopniu ułatwiłyby pracę:
-> w eksporcie punktów są dwa razy współrzędne y (pominięty został x)
-> możliwość eksportu punktów z całego GML (przy większych opracowaniach pojedyncze klikanie jest dosyć uciążliwe)
-> możliwość eksportu razem z punktami i atrybutami, numeru w operacie archiwalnym i numeru operatu technicznego z którego pochodzi (ułatwiłoby to analizę)
-> powierzchnia działek i użytków (miejsca po przecinku) GML ma informacje czy powierzchnia jest do ara czy do metra, można byłoby to wykorzystać i podawać już poprawne powierzchnie ewidencyjne (nie trzeba byłoby tego sprawdzać w wypisach)
-> rozdzielenie adresu na kolumny: kod, miejscowość, ulicę z numerem 
Poza tym daje dużego kciuka w górę i widzę duży potencjał :)

**Reply by KacperCe** [reply]
Witam, dzięki na wstępie @RybarskiDominik  za sprawne poprawienie znalezionych bugów. 
-> adres jako osobne kolumny moim zdaniem super pomysł, popieram (szczególnie przydatne w przypadku ulicy i numeru bud./lokalu
Tak naprawdę każdy z tych wpisów mógłby być osobną kolumną, dużo łatwiej jest sobie to później złączyć w excelu jeżeli ktoś potrzebuje niż rozdzielanie jednej kolumny z adresem na osobne kolumny (tutaj problem się pojawia przy wielowyrazowych nazwach ulic i miejscowości)
![image](https://github.com/user-attachments/assets/dcbe5835-9434-4d8f-9a03-30c8ed9413fd)

**Reply by RybarskiDominik** [reply]
Zrobione:
-> w eksporcie punktów są dwa razy współrzędne y (pominięty został x)
-> możliwość eksportu punktów z całego GML (przy większych opracowaniach pojedyncze klikanie jest dosyć uciążliwe)
-> powierzchnia działek i użytków (miejsca po przecinku) GML ma informacje czy powierzchnia jest do ara czy do metra, można byłoby to wykorzystać i podawać już poprawne powierzchnie ewidencyjne (nie trzeba byłoby tego sprawdzać w wypisach)

Potrzebna większa zmiany kodu:
-> Export punktów (Dedykowane okno)
->możliwość eksportu razem z punktami i atrybutami, numeru w operacie archiwalnym i numeru operatu technicznego z którego pochodzi (ułatwiłoby to analizę)
-> adres jako osobne kolumny
-> Menu ustawienia.

https://github.com/RybarskiDominik/GML-2021/releases/tag/1.5.5

**Reply by Kamila111111** [reply]
Dziękuje poprawione funkcje działają :)

**Reply by Kamila111111** [reply]
Mam jeszcze jedno spostrzeżenie dotyczące eksportu punktów, przy pracach gdzie jest kilka obrębów numer punktu nie wystarcza potrzebny jest jego identyfikator, aby uniknąć pomyłek i powieleń może warto zamiast numeru zastosować identyfikator punktu?

**Reply by RybarskiDominik** [reply]
Będzie taka opcja w ustawieniach za niedługo.

**Reply by RybarskiDominik** [reply]
Już jest.
